src/doing/pr/create_pr.py

```diff
@@ -77,8 +77,12 @@ def cmd_create_pr(
     )
     remote_branches = [x.rpartition("/")[2] for x in remote_branches if x.startswith("refs/heads")]
 
+    # Find the default branch from which to create a new branch and target the pull request to
+    cmd = f"az repos show --repository '{repo_name}' --org '{organization}' -p '{project}'"
+    default_branch = run_command(cmd)["defaultBranch"]
```

Comment:
Are we certain this always has a value? Let's add a fallback:

```python
run_command(cmd)["defaultBranch"]
```

with

```python
run_command(cmd).get("defaultBranch","refs/heads/master") 
```

Reply:
fixed